docs/architecture/psa-migration/outcome-analysis.sh

```diff
@@ -0,0 +1,96 @@
+#!/bin/sh
+
+# This is only an example/template script, you should make a copy and edit it
```

Comment:
A description of this script would be useful.

Reply:
Also - maybe a different name, as the commit message suggests? `print-coverage-comparison.sh` or something about regression?

Reply:
I added as description and some usage notes.

I agree that the name you're suggesting is better, as it describes _what_ the script does, while the current name describes the _how_ (analyzing the outcomes files), which is indeed not great. However, I had already referenced this script from several issue descriptions created in the meantime, so it would be convenient to avoid renaming. I think since the script is only for internal use, and not really intended to be discovered just by looking at file names (but rather from the strategy document, or an issue description that references it), we can live with a sub-optimal name, now that we at least have a proper description inside the file. Is that OK for you?